src/runtime_src/doc/toc/cloud_vendor_support.rst

```diff
@@ -191,3 +191,14 @@ The following picture illustrates how XRT is being deployed in different scenari
 
 .. image:: xrt-deployment-cloud.svg
    :align: center
+
+Special Case
+============
+
+There is special case where download xclbin from within user VM is not required.
+
+In this special case, neither MSD/MPD nor plugins are required since there is no real xclbin download happening. xclbins can be preloaded
+either by hypervisor or by a service domain (VM) where the mgmt PF is assigned. The apps in user VM are running without any change, ie
+xclbin download ioctl is still issued to xocl driver, xocl driver gets the uuid of the running xclbin with a XCL_MAILBOX_REQ_PEER_DATA
+mailbox opcode to xclmgmt, and if the uuid matches the one of the xclbin being downloaded, the ioctl returns success without real xclbin
```

Comment:
the one of the xclbin being downloaded, the ioctl returns success  --> with that of the xclbin requested for download, the ioctl returns immediately with success. If the uuids do not match download request in the guest fails.

Reply:
changed